src/lightning/app/core/queues.py

```diff
@@ -388,19 +388,30 @@ def get(self, timeout: Optional[float] = None) -> Any:
         if timeout is None:
             while True:
                 try:
-                    return self._get()
+                    try:
+                        return self._get()
+                    except requests.exceptions.HTTPError:
+                        pass
                 except queue.Empty:
                     time.sleep(HTTP_QUEUE_REFRESH_INTERVAL)
 
         # make one request and return the result
         if timeout == 0:
-            return self._get()
+            try:
+                return self._get()
+            except requests.exceptions.HTTPError:
```

Comment:
can you pls add a note about why this error happens and it is fine to ignore it?